content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -279,7 +338,7 @@ async function handleEvent(event: FetchEvent): Promise<Response> {
           headers: JSON.stringify(request.headers),
         },
       },
-    })
+    }).then()
```

Comment:
Wouldn't we usually just `await prisma.log.create...`? @millsp

Reply:
@janpio The background for this is here: https://www.notion.so/prismaio/MongoDB-Connections-strings-1bb2aa956bb74fe7ae5f44ffcd0af84e#84d050b8c9dc4751aba8a31114ef3a6a

I can change it if the `await prisma.log.create...` approach makes more sense here.

Reply:
More context: https://prisma-company.slack.com/archives/CUXLS0Z6K/p1659522110973269

Reply:
Yeah, that explain why we have to either `await` or `.then()` the "Promise".